egs/swbd/s5c/local/rnnlm/tuning/run_lstm_f.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
```

Comment:
please rename this to 1a.  I prefer to start from there for new things.  There is no need for consistency with your private experiments.